lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1140,6 +1140,14 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'If `true`, disables Action Cable instrumentation.'
         },
+        :disable_action_dispatch => {
+          :default => false,
+          :public => true,
+          :type => Boolean,
+          :dynamic_name => true,
```

Comment:
A rogue dynamic name!

Reply:
That shows how long I've had this branch! It was prepped before we got to the bottom of not wanting this key/val pair and I missed it when coming back to it. Thank you! Addressed with [418c017](https://github.com/newrelic/newrelic-ruby-agent/pull/1745/commits/418c017c7434c2bf27e4294bc3028410cf997e96)